pipelines/pingcap/tidb-engine-ext/latest/ghpr_unit_test.groovy

```diff
@@ -0,0 +1,85 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+// should triggerd for master and release-6.2.x branches
+final K8S_NAMESPACE = "jenkins-tiflash"
```

Comment:
please add a git ops dir in [this folder](https://github.com/PingCAP-QE/ee-ops/tree/main/apps/prod/jenkins/post) later.

Reply:
I will add it later